deps/spidershim/src/v8functiontemplate.cc

```diff
@@ -417,31 +417,19 @@ bool FunctionTemplate::HasInstance(Local<Value> val) {
   Isolate* isolate = Isolate::GetCurrent();
   JSContext* cx = JSContextFromIsolate(isolate);
   AutoJSAPI jsAPI(cx, this);
```

Comment:
You should probably ensure that obj is wrapped into our compartment at this point or something.  Otherwise we can have situations where we're coming in on some compartment different from the one of `this` and the arg is actually a cross-compartment wrapper for something that _is_ an instance of this template... but a CCW will test false for IsObjectFromTemplate.